extractor/extractor.go

```diff
@@ -111,6 +111,39 @@ func ExtractWithFlags(buildFlags []string, patterns []string) error {
 
 	wg.Wait()
 
+	cwd, err := os.Getwd()
+	if err != nil {
+		log.Printf("Warning: unable to get working directory: %s", err.Error())
+		log.Println("Skipping go.mod extraction")
+	}
+	rcwd, err := filepath.EvalSymlinks(cwd)
+	if err == nil {
+		cwd = rcwd
+	}
+
+	log.Printf("Walking file tree from %s to discover go.mod files...", cwd)
+
+	filepath.Walk(cwd, func(path string, info os.FileInfo, err error) error {
+		if filepath.Base(path) == "go.mod" {
```

Comment:
```suggestion
		if filepath.Base(path) == "go.mod" && info != nil && info.Mode.IsRegular() {
```

or at least `&& !info.IsDir()`